client/src/main.js

```diff
@@ -1,6 +1,6 @@
 var nunjucks = require('nunjucks');
 var fs = require('fs');
-var teams = require('../data/teamsData.js');
+var teams = require('./data/teamsData.js');
```

Comment:
We changed the directory the file is in, so we need to change the relative path to the data.